File: python/algoritmos_basicos/primos.py
def primosI(n):
    l = []
    for i in range(1,n + 1):
        if n % i == 0:
            l.append(i)
    
    if len(l) > 2:
        print(f"o número {n} não é primo, ele é divisivel por : {l}")
    else:
        print(f"o número {n}  é primo, ele é divisivel por : {l}")



primosI(856)

File: c++/Algoritmos Básicos/primos.cpp
#include <iostream>
using namespace std;

int primosI(int n)
{
    int divisores = 0;

    for (int i = 1; i <= n; i++)
    {
        if (n % i == 0)
        {
            divisores++;
        }
    }

    if (divisores == 2)
    {
        cout << "o número " << n << " é primo" << endl;
    }
    else
    {
        cout << "o número " << n << " não é primo" << endl;
    }

    return 0; 
}

int main()
{
    primosI(856); 
    return 0;
}
